src/bandersnatch/master.py

```diff
@@ -52,6 +52,15 @@ def __init__(
             logger.error(err)
             raise ValueError(err)
 
+    def _build_proxy_connector(self) -> ProxyConnector | None:
+        """Build a ProxyConnector if a proxy is set in the config"""
+        if not self.proxy:
```

Comment:
Do we verify self.proxy is valid proxy URI aywhere or just leave that to when we go to use it via socks_proxy or aiohttp http proxy support?
- Just food for thought here - Wondering if we should do better ...